bigchaindb/tendermint/lib.py

```diff
@@ -53,6 +53,7 @@ def write_transaction(self, transaction, mode):
     def store_transaction(self, transaction):
         """Store a valid transaction to the transactions collection."""
 
+        self.update_utxoset(transaction)
```

Comment:
I don't think `store_transaction` is being used to store transactions anymore. Since we migrated to storing transactions on `commit` so `store_bulk_transactions` is now used instead.

Reply:
Ah ok! Will make necessary changes.

Reply:
@kansi The change was made in b254b303bac70e7b7829385585a8c685c0016c1a.